frontend/packages/dev-console/integration-tests/support/pages/add-flow/git-page.ts

```diff
@@ -235,15 +235,7 @@ export const gitPage = {
     cy.get(gitPO.gitSection.validatedMessage)
       .should('not.include.text', 'Validating...')
       .and('not.include.text', messages.addFlow.buildDeployMessage);
-    cy.get('body').then(($body) => {
-      if (
-        $body.text().includes(messages.addFlow.rateLimitExceeded) ||
-        $body.find('[aria-label="Warning Alert"]').length ||
-        $body.text().includes(messages.addFlow.privateGitRepoMessage)
-      ) {
-        gitPage.selectBuilderImageForGitUrl(gitUrl);
-      }
-    });
```

Comment:
This code is needed, because if we get rate limitation, then only this line ("gitPage.selectBuilderImageForGitUrl(gitUrl);") should execute

Reply:
My idea is that we call `selectBuilderImageForGitUrl` always so that we notice if the test breaks again in the future. But when you don't like the idea I can restore that we call this only in this edge case which mostly happen on the CI server (only)

Reply:
Reverted this as it is not required for our fix. Thanks for the quick feedback.